sbi/diagnostics/sbc.py

```diff
@@ -89,7 +79,7 @@ def run_sbc(
             stacklevel=2,
         )
 
-    # Get posterior samples, batched or parallelized.
+    # Get posterior samples, batched or parallelized
```

Comment:
I prefer sentence-case comments (starting with capital letter and ending with fullstop).